tests/unit/amo/components/TestSearch.js

```diff
@@ -433,6 +435,24 @@ describe(__filename, () => {
     expect(wrapper.find(NotFound)).toHaveLength(1);
   });
 
+  it('renders a robots meta when there is no results', () => {
+    const { store } = dispatchSearchResults({ addons: [] });
+    const root = renderWithStore({ store });
```

Comment:
Minor improvement because we shouldn't set the `count` manually. It is a redux prop (injected).